src/java/io/bazel/rulesscala/coverage/instrumenter/JacocoInstrumenter.java

```diff
@@ -98,6 +99,13 @@ private FileVisitResult actuallyVisitFile(Path inPath, BasicFileAttributes attrs
                 } else {
                     Files.copy(inPath, outPath);
                 }
+
+                if(outPath.toString().endsWith(".class")) { 
```

Comment:
isn't this condition always true when `inPath.toString().endsWith(".class")`? If so why add another condition and not add it to the above? Probably best to extract the current code there to a private method and also the body of the `then` here to a private method for clarity.

Reply:
This should be moot now.